### DISCUSSION TITLE
简化首页增加删除滚动推荐

### DISCUSSION THREAD

**Discussion by miku1958** [the1812/Bilibili-Evolved#4578] [功能建议 / Ideas]
能否给简化首页添加删除这个滚动推荐的功能谢谢!
<img width="606" alt="image" src="https://github.com/the1812/Bilibili-Evolved/assets/24806909/e09771b0-815a-4244-a6a0-30a8bfcf9490">

**Comment by miku1958** [comment]
它的class name应该是"recommended-swipe grid-anchor"

**Comment by miku1958** [comment]
我直接删掉它的话, 首页布局会有点错位

**Comment by Geeyun-JY3** [comment]
`简化首页`这组件在新版首页基本上已经失效了。今天实测好像只有`分区栏`能成功屏蔽。

**Reply by the1812** [reply]
又出啥新版主页了? 我好像没遇到, 现在主页还是那个无限滚动的版本

**Reply by Geeyun-JY3** [reply]
我说的就是这个无限滚动的新版主页啊。
在新版首页也没显示多少选项了：
![image](https://github.com/the1812/Bilibili-Evolved/assets/49396421/59ae9f16-35e3-429a-8ae2-1da149dd4ca4)
